cmds/http-gateway/main.go

```diff
@@ -176,16 +179,37 @@ func myHTTPError(ctx context.Context, mux *runtime.ServeMux, marshaler runtime.M
 	}
 	w.Header().Set("Content-Type", contentType)
 
```

Comment:
Fix func comment: "... only needed to add 1 extra Code to handle but since they didn't..."

Reply:
Will do.

Reply:
Done.